server/routes/trees/id.js

```diff
@@ -0,0 +1,67 @@
+/* ALERT ALERT If you change anything here, change it in wtt_area id.js as well!!! */
```

Comment:
Hmm.  Can this code be extracted into a separate package that both the server and area codebases import?  Or can wtt_area be merged into the server?  Not really sure what that repo does, or if it needs to be separate.